packages/builders/src/components/ActionRow.ts

```diff
@@ -3,14 +3,16 @@ import type { ButtonComponent, SelectMenuComponent } from '..';
 import type { Component } from './Component';
 import { createComponent } from './Components';
 
+export type MessageComponent = ActionRowComponent | ActionRow;
+
 export type ActionRowComponent = ButtonComponent | SelectMenuComponent;
 
 // TODO: Add valid form component types
 
 /**
  * Represents an action row component
  */
-export class ActionRow<T extends ActionRowComponent> implements Component {
+export class ActionRow<T extends ActionRowComponent = ActionRowComponent> implements Component {
```

Comment:
Out of scope?

Reply:
Eh maybe? I explained in the description it was just a small thing to make it easier to type but I can revert if you want